DVTShowcase/HomeViewController.swift

```diff
@@ -39,29 +32,48 @@ class HomeViewController: DVTShowcaseViewController, UITableViewDelegate, UITabl
                 self.showcaseAppArray = showcaseAppArray
                 DispatchQueue.main.async(execute: { () -> Void in
                     self.showcaseAppTableView.reloadData()
-//                    self.hideLoading()
                 })
             } else {
-//                DispatchQueue.main.async(execute: { () -> Void in
-//                    self.hideLoading()
-//                })
                 self.showAlert(title: "Error!", message: message!)
             }
         }
     }
     
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    func setUpTableView() {
+        self.showcaseAppTableView.delegate = self
+        self.showcaseAppTableView.dataSource = self
+        self.showcaseAppTableView.rowHeight = UITableViewAutomaticDimension
+        self.showcaseAppTableView.estimatedRowHeight = 130
+        self.showcaseAppTableView.register(UINib.init(nibName: String(describing: ShowcaseAppTableViewCell.self), bundle: nil), forCellReuseIdentifier: "CellIdentifier")
+    }
+    
+    private func setUpNavigationBar() {
+        let narBArImageView = UIImageView(frame: CGRect(x: 0, y: 0, width: 30, height: 40))
+        narBArImageView.image = UIImage(named: "dvt_icon")
+        self.navigationItem.title = "DVT App Showcase"
+    }
+    
+    
+}
+
+extension HomeViewController: UITableViewDelegate, UITableViewDataSource {
+    
     func numberOfSections(in tableView: UITableView) -> Int {
-        return showcaseAppArray.count
+        return 1
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return 1
+        return  showcaseAppArray.count
```

Comment:
this naming is bad, please change to something like showcaseApps, we know its an array by its type dont have to mention it in the name of the variable